modules/api/src/main/java/org/apache/ignite/lang/ColumnNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.ignite.lang;
+
+/**
+ * Exception is thrown when appropriate column is not found.
+ */
+public class ColumnNotFoundException extends IgniteException {
+    /**
+     * Create a new exception with given column name.
+     *
+     * @param name Column name.
+     * @param fullName Schema and table name.
```

Comment:
```suggestion
     * @param tableName Table canonical name.
```

Reply:
"Schema and table name." doen't imply any format,
while canonical name it concated names with '.' dot-notation: "MySchema.MyTable".
Or am I wrong?